src/app/portlets/dot-edit-page/layout/dot-edit-layout-designer/dot-edit-layout-designer.component.html

```diff
@@ -1,10 +1,10 @@
 <form [formGroup]="form" (ngSubmit)="saveLayout()" novalidate class="dot-edit-layout__form">
     <div class="dot-edit-layout__toolbar">
```

Comment:
This whole ```div``` should be a component, I didn't do it right now because of the ```<dot-layout-properties [group]="form.get('layout')"></dot-layout-properties>``` that handle the ```form.get('layout')``` and I didn't want to overcomplicate this issue, also I think that this properties shouldn't be in the toolbar so we might have to relocate that in a possible future redesign.